From 99d71f6e2e58d63b8457fff5dbf168e1e40ba8a6 Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Thu, 25 Apr 2024 12:14:16 +0300
Subject: [PATCH] [SW-15774] package: zephyr: remove dead code

Zephyr packages not include the agent build any more therefore
all related code is not used anymore, and can be removed.

Change-Id: I2c2172219b385ddb141c56cf680b03aa582e88b3
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/package/zephyr/zephyr.mk b/package/zephyr/zephyr.mk
index 1717450..81b53a4 100644
--- a/package/zephyr/zephyr.mk
+++ b/package/zephyr/zephyr.mk
@@ -125,34 +125,6 @@
 ZEPHYR_COMPONENTS_LIST += agent-fwl
 endif
 
-ifdef BR2_arcle
-ZEPHYR_MARGIN_TOOL_AGENT_BOARD = hl_$(DEVICE_NAME)_agent_$(HL_TARGET)
-else
-ZEPHYR_MARGIN_TOOL_AGENT_BOARD = hl_$(DEVICE_NAME)_$(HL_TARGET)
-endif
-
-# first argument is agent name, second argument is additional configuration opts
-define CREATE_ZEPHYR_MARGIN_TOOL_AGENT
-$(eval AGENT_LOWER_CASE = $(strip $1)) \
-$(eval ZEPHYR_COMPONENTS_LIST += $(AGENT_LOWER_CASE)) \
-$(eval AGENT_UPPER_CASE = $(shell echo $(AGENT_LOWER_CASE) | tr a-z A-Z)) \
-$(eval AGENT_UPPER_CASE = $(strip $(AGENT_UPPER_CASE))) \
-$(eval ZEPHYR_AGENT_DIR = build-agent-$1) \
-$(eval ZEPHYR_$(AGENT_UPPER_CASE)_DIR = build-agent-$(AGENT_LOWER_CASE)) \
-$(eval ZEPHYR_$(AGENT_UPPER_CASE)_NAME = zephyr_agent_$(AGENT_LOWER_CASE)) \
-$(eval ZEPHYR_CONF_DEFAULT ?= $(ZEPHYR_AGENT_DIR)) \
-$(eval ZEPHYR_CONF_OPTS_AGENT = -DBOARD=$(ZEPHYR_MARGIN_TOOL_AGENT_BOARD)) \
-$(eval ZEPHYR_CONF_OPTS_AGENT += -DDEVICE=$(DEVICE_NAME)) \
-$(eval ZEPHYR_$(AGENT_UPPER_CASE)_SRCDIR = $(EMBEDDED_DIR)/zephyr-plugins/margin-tools/$(AGENT_LOWER_CASE)) \
-$(eval ZEPHYR_CONF_OPTS_AGENT += $(ZEPHYR_$(AGENT_UPPER_CASE)_SRCDIR)) \
-$(eval ZEPHYR_CONF_OPTS_AGENT += $2) \
-$(eval ZEPHYR_CONFIGURE_CMDS += && $$(ZEPHYR_CONF_ENV) $(HOST_DIR)/bin/cmake $$(ZEPHYR_CONF_OPTS) $(ZEPHYR_CONF_OPTS_AGENT) -B $(ZEPHYR_AGENT_DIR)) \
-$(eval ZEPHYR_BUILD_CMDS += && $(TARGET_MAKE_ENV) $(MAKE) -C $$(@D)/$(ZEPHYR_AGENT_DIR)) \
-$(eval ZEPHYR_INSTALL_TARGET_CMDS += && $(INSTALL) -D -m 0644 $$(@D)/$(ZEPHYR_AGENT_DIR)/zephyr/zephyr.bin $(STAGE_DIR)/zephyr_agent_$1.bin) \
-$(eval ZEPHYR_INSTALL_TARGET_CMDS += && $(INSTALL) -D -m 0644 $$(@D)/$(ZEPHYR_AGENT_DIR)/zephyr/zephyr.stat $(STATS_DIR)/zephyr_agent_$1.stat) \
-$(foreach module,$(ZEPHYR_$(AGENT_UPPER_CASE)_MODULES),$(call DEFINE_MODULE_ENVVARS,$(module),$(ZEPHYR_$(AGENT_UPPER_CASE)_DIR), $(ZEPHYR_$(AGENT_UPPER_CASE)_NAME)))
-endef
-
 ifdef BR2_PACKAGE_AGENT_HBM
 ZEPHYR_COMPONENTS_LIST += agent-hbm
 endif
